.github/workflows/prettier.yml

```
@@ -10,13 +10,17 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: '14'
+      - run: npm ci
+      - name: Fail CI
```

The name does not make sense, CI will fail if a task fails

```suggestion
      - name: Prettier
```